### ISSUE TITLE
Log in via social providers

### ISSUE THREAD

**Issue by Jandev** [jfversluis/CfpExchange#56]
I would love to have the ability to log in via some of the social providers.

No need to have local accounts (accounts not created via a social provider) in my opinion, because that's just a lot of hassle. Only thing which needs to be stored in the CFP Exchange repository is the `Name` an identifier and maybe the `E-mail address` if you want to send out mailings later on (take into consideration the GDPR before sending out those mails).

The userbase (at the moment) are all developers/IT-pro's, so the GitHub & Twitter social providers should suffice.
Of course adding Google & Microsoft accounts is doable also.

Some documentation or .NET Core 2.1 integration:
https://docs.microsoft.com/en-us/aspnet/core/security/authentication/social/?view=aspnetcore-2.1
Covers all major providers

**Comment by jfversluis** [comment]
Agree! @wmeints had something in the works, but I think this was the other way around, choosing local accounts 😄 

I don't really care which one it will be, although social login makes the most sense in terms of not having to maintain all kinds of plumbing there yourself.
